### ISSUE TITLE
Menus don't work right in certain DVDs

### ISSUE THREAD

**Issue by balr0g** [gmarty/DVD.js#4]
The menus in certain DVDs such as "Sita Sings the Blues" seem to be off, and clicking on a menu option doesn't necessarily do what it's supposed to.

Is this something you can reproduce on your end?

**Comment by gmarty** [comment]
Yes, that's because not all VM opcodes are implemented. So basically, unimplemented opcodes use 'console.log' like https://github.com/gmarty/DVD.js/blob/converter/src/vm/recompile.ts#L489

Also, regarding Sita official DVD, I know there is a bug somewhere. I tried hard to find what's wrong there but couldn't find anything. I should have another look though.

**Comment by balr0g** [comment]
Maybe having a compatibility list would be a good idea?

I'll likely be working with DVDs that come from odd sources ("homemade" or low-budget DVDs on Internet Archive among others) so things might or might not work.

**Comment by Blamsoft** [comment]
I'm seeing a similar problem with unimplemented opcodes. I have a common DVD, The Lord of the Rings: The Fellowship of the Ring for the test case. Would you be interested in receiving funding to improve the VM opcodes or working together to improve DVD.js to handle popular DVDs?
